### DISCUSSION TITLE
可以請你分享Win10 pro裡的三角形鼠標嗎?

### DISCUSSION THREAD

**Discussion by mouse723** [WhatTheBlock/WindowsSimplify#40] [General]
可以請你分享Win10 pro裡的三角形鼠標嗎?
謝謝，謝謝，謝謝

**Comment by WhatTheBlock** [comment]
這是recitfy11 v2.5的版本，我有空再打包整套的美化版鼠標

**Reply by mouse723** [reply]
真是太棒了，非常感謝，將會持續關注好作品

**Comment by WhatTheBlock** [comment]
[Rectified v2.5 Cursors](https://github.com/WhatTheBlock/WindowsSimplify/releases/download/utils/Rectified_v2.5_Cursors.rar)

**Reply by mouse723** [reply]
真是萬分感謝您
